tests/js/jest.config.json

```diff
@@ -12,6 +12,7 @@
 		"@woocommerce/atomic-utils": "assets/js/atomic/utils",
 		"@woocommerce/icons": "assets/js/icons",
 		"@woocommerce/settings": "assets/js/settings/shared",
+		"@woocommerce/blocks(.*)$": "assets/js/blocks/$1",
```

Comment:
@dinhtungdu this is breaking module name mapper, I think because blocks(.*) probably matches some other existing packages. for example it would match `@woocommerce/blocks-registry` which is already listed further down.

Reply:
Nice catch! I will update the pattern.

> This is one of the many reasons I really hate aliases fwiw. This kinda stuff is not maintainable just to save some ../ that gets autofilled by the editor.

Maybe another RFC 😄 ?